src/components/Proposal/View/Budget/RequestedBudgetCard.tsx

```diff
@@ -82,10 +82,11 @@ export default function RequestedBudgetCard({ proposal, budget }: Props) {
   }, [
     isProposalActive,
     allocatedCategoryBudget,
+    isOverBudget,
+    categoryBudget,
     requestedBudget,
     remainingBudgetDisplayed,
-    categoryBudget,
-    isOverBudget,
+    totalCategoryBudget,
```

Comment:
Are `totalCategoryBudget` and `allocatedCategoryBudget` necesary in the deps if we already have `categoryBudget`?